### PR TITLE
27 hashes

### PR DESCRIPTION
**Pull request by niksseif**
# Hash lesson

### Does this PR close any issues?
*This PR closes.[#27](https://github.com/turingschool/curriculum-site/issues/27)*

### Description

- [x] Added JavaScript examples alongside the existing Ruby examples.
- [x] Updated explanations and instructions to be language-neutral where applicable.
- [x] Revised vocabulary and concepts to apply to both Ruby and JavaScript.
- [x] Ensured that all practice exercises include solutions in both Ruby and JavaScript.

### CODE REVIEW

File: module1/lessons/hashes.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
 ---
 title: Hashes
+subheading: lesson is Hashes
```
Comment (KatBrandt) [inline module1/lessons/hashes.md:3]: Remove this line, it shows up on the actual lesson plan. Or change it to something like: `Ruby data structure`, or just omit.

File: module1/lessons/hashes.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 title: Hashes
+subheading: lesson is Hashes (or Objects)
 layout: page
 ---
 
 ### Learning Goals
-- Identify the differences between Hashes and Arrays. 
-- Practice building, accessing, and updating Hashes.
+- Identify the differences between Hashes(or Objects in Javascript) and Arrays. 
+- Practice building, accessing, and updating Hashes(or Objects).
```
[the remark this answers is not in the document]
Comment (niksseif) [reply module1/lessons/hashes.md]: Fixed it here [ba6d306](https://github.com/turingschool/curriculum-site/pull/59/commits/ba6d306ac862053a9d41248109c8c5670ba31e02)

File: module1/lessons/hashes.md [hunk 3]
```diff
@@ -35,16 +49,18 @@ Take note of:
...
+  - What syntax did you learn for hashes ?
   - What questions came up while you were working and how did you get those answered?
   - What questions do you still have?
   - Anything else that seems important.
 
 Resources to use while working through the exercises:
   - Your breakout room mates!
-  - Google (ex: `Ruby how to make a new hash with a default value`)
+  - Google (ex: Ruby how to make a new hash with a default value, JavaScript how to make a new object with a default value)
   - [Ruby Docs](https://ruby-doc.org/core-2.7.0/Hash.html)
```
Comment (KatBrandt) [inline module1/lessons/hashes.md:59]: Need to update this link to the current version of Ruby we're using. This one is for version 2.7.0. Can just change the 2.7.0 to 3.0.0 in this link and that will work.

File: module1/lessons/hashes.md [hunk 4]
```diff
@@ -77,14 +93,39 @@ Ex:
...
+</section>
 
 
 ### What data type can a key be?
   
 Keys of a hash can be any data type. 
-Often, we see them as symbols.
-A symbol is an immutable data-type in Ruby. They will start with a `:` (colon).
-Keys can also be seen commonly as strings, however they can be integers as well.
+Often, we see them as symbols (in Ruby) or strings (in Javascript).However, they can be integers as well.
```
Comment (KatBrandt) [inline module1/lessons/hashes.md:117]: Add a space after the period, before the word `However`

File: module1/lessons/hashes.md (hunk 3 above)
Comment (niksseif) [reply module1/lessons/hashes.md]: Thank you 🙏🏼 - fixed it here [3be59e3](https://github.com/turingschool/curriculum-site/pull/59/commits/3be59e36e8a0380988a881c11c13f66fd4be31c1)

File: module1/lessons/hashes.md (hunk 4 above)
Comment (niksseif) [reply module1/lessons/hashes.md]: Fixed it here [cb03455](https://github.com/turingschool/curriculum-site/pull/59/commits/cb03455e135e37994804f34ee67328c71214bbd5)

**Comment by niksseif** [comment]
Thank you, @KatBrandt, for reviewing this PR. Everything should be fixed now.

**Pull request merged**